buildit_config:
    name: grizzly_base
    robot_description:
        package: 'grizzly_description'
        model: 'urdf/base.urdf.xacro'
    mount_points:
        sensor_anchor_link:
            position:
                x: 1.0
                y: 1.0
                z: 1.0
            orientation:
                r: 0.5
                p: 0.0
                y: 0.0
    edit_positions: false
    edit_orientation: false
    modify_model: false
    

